app/admin/resources/about_us.ts

```diff
@@ -0,0 +1,29 @@
+import { LucidResource } from "@adminjs/adonis";
+
+import { linkTypeEnumsValues } from "#enums/link_type";
+import AboutUsGeneral from "#models/about_us_general";
+import AboutUsGeneralLink from "#models/about_us_general_link";
+
+import { readOnlyTimestamps } from "./utils/timestamps.js";
+
+export const aboutUsResource = {
+  resource: new LucidResource(AboutUsGeneral, "postgres"),
+  options: {
+    properties: {
+      id: {
+        isVisible: false,
+      },
+      ...readOnlyTimestamps,
+    },
+  },
+};
```

Comment:
Actions such as `new`, `delete` and `bulkDelete` should be disabled for this resource, as there can only ever be one of them, it should never be deleted and there's a production seeder that creates the first instance automatically.
```diff
 export const aboutUsResource = {
   resource: new LucidResource(AboutUsGeneral, "postgres"),
   options: {
     properties: {
       id: {
         isVisible: false,
       },
       ...readOnlyTimestamps,
     },
+    actions: {
+      new: {
+        isAccessible: false,
+        isVisible: false,
+      },
+      delete: {
+        isAccessible: false,
+        isVisible: false,
+      },
+      bulkDelete: {
+        isAccessible: false,
+        isVisible: false,
+      },
+    },
   },
 };
```